### PR TITLE
[4.0] Installation of the language pack was successful

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #26288 

### Summary of Changes
Corrects the success message when installing a language from the Extensions: Languages page (`administrator/index.php?option=com_installer&view=languages`)

### Testing Instructions
Install or reinstall the french language

### Expected result
Message stating that the language pack has been installed
image

### Actual result
Message stating that something called a package has been installed.

### Documentation Changes Required
None that I am aware of

### CODE REVIEW

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 58177de4c8938a100bbaf8dfed4e83abb37c3ef2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26310">issues.joomla.org/tracker/joomla-cms/26310</a>.</sub>

File: administrator/components/com_installer/Model/InstallModel.php [hunk 1]
```diff
@@ -207,7 +208,7 @@ public function install()
 		else
 		{
 			// Package installed successfully.
-			$msg = Text::sprintf('COM_INSTALLER_INSTALL_SUCCESS', Text::_('COM_INSTALLER_TYPE_TYPE_' . strtoupper($package['type'])));
+			$msg = Text::sprintf('COM_INSTALLER_INSTALL_SUCCESS', Text::_('COM_INSTALLER_TYPE_TYPE_' . strtoupper($installLang) . strtoupper($package['type'])));
```
Comment (Quy) [inline administrator/components/com_installer/Model/InstallModel.php]:
```suggestion
			$msg = Text::sprintf('COM_INSTALLER_INSTALL_SUCCESS', Text::_('COM_INSTALLER_TYPE_TYPE_' . strtoupper($installLang . $package['type'])));
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 272744e6727eb0dac90428d20ce711105838f397<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26310">issues.joomla.org/tracker/joomla-cms/26310</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 272744e6727eb0dac90428d20ce711105838f397<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26310">issues.joomla.org/tracker/joomla-cms/26310</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26310">issues.joomla.org/tracker/joomla-cms/26310</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**